app-text/xhtml11/xhtml11-20101123.ebuild

```diff
@@ -45,9 +45,9 @@ pkg_postinst() {
 	einfo "Installing xhtml11 in the global XML catalog"
 	"${XMLTOOL}" --noout --add 'public' '-//W3C//DTD XHTML 1.1//EN' \
 		"${DTDDIR}"/xhtml11-flat.dtd "${CATALOG}"
-	"${XMLTOOL}" --noout --add 'rewriteSystem' 'http://www.w3.org/TR/xhtml11/DTD' \
```

Comment:
Honestly, I have no clue whether it's fine or not.

@ulm, perhaps you know?

Reply:
IIUC, the DTD must be the exact string from the spec, so you cannot replace `http` by `https`:
https://www.w3.org/QA/2002/04/valid-dtd-list.html

Also, it is only a unique identifier and not necessarily a real location. Indeed, there's nothing at the http://www.w3.org/TR/xhtml11/DTD target URL (and nothing at the https URL either).

Reply:
@ulm 
Thanks for the clarification, i've removed the changes.